### PR TITLE
Collections 0.2.0

### PR DESCRIPTION
**Pull request by julienrf**

### CODE REVIEW

File: blog/_posts/2017-06-30-collections-0.2.0.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+post-type: blog
+by: Julien Richard-Foy
+title: Redesigned collections v0.2.0
+---
+
+We are happy to announce the
+[0.2.0 release](https://index.scala-lang.org/scala/collection-strawman/collection-strawman/0.2.0)
+of the redesigned collections!
+
+You can read more about our effort to redesign the standard collections in this previous
```
Comment (heathermiller) [inline blog/_posts/2017-06-30-collections-0.2.0.md]: Summarize what the new collections are about. Don’t point people to read a whole new document. You can make a summary in 3-4 sentences, easy.

File: blog/_posts/2017-06-30-collections-0.2.0.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+[blog post](http://www.scala-lang.org/blog/2017/02/28/collections-rework.html).
+
+## Current status
```
Comment (heathermiller) [inline blog/_posts/2017-06-30-collections-0.2.0.md]: Why not say that you wish to publish monthly milestones?

File: blog/_posts/2017-06-30-collections-0.2.0.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+The goal of this release is to provide a drop-in replacement of the current collections
```
Comment (heathermiller) [inline blog/_posts/2017-06-30-collections-0.2.0.md]: Isn't that the general goal of the entire collections project? Not this release specifically...

File: blog/_posts/2017-06-30-collections-0.2.0.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+for most use cases: the main collection types (`Seq`, `Set` and `Map`, mutable or
+immutable, sorted or not sorted) are implemented
+and support the most common operations. You can see a detailed list of the current
+implementation status
+[here](https://github.com/scala/collection-strawman#implemented-collection-types).
```
Comment (heathermiller) [inline blog/_posts/2017-06-30-collections-0.2.0.md]: Again, don't link to another document. People came to this page to see the status. They don't want to be linked to 5 other pages. Try to make this more self-contained. You can probably easily make a list of implemented collection types and put them in the blog article, no?

File: blog/_posts/2017-06-30-collections-0.2.0.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+To use it in your project, add the following dependency:
```
Comment (heathermiller) [inline blog/_posts/2017-06-30-collections-0.2.0.md]: Why not say instead, "To experiment with the latest milestone of the new collections library, just add the following dependency:"

(You don't necessarily want people using this library in production or anything, right? "Use in your project" makes it sound like you do)

File: blog/_posts/2017-06-30-collections-0.2.0.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+import strawman.collection.immutable.List
+
+println(List(1, 2, 3).map(x => x + 1))
+~~~
+
+In order to make this release a real “drop-in” replacement of the current collections,
+we are working on a
+[rewrite tool](https://github.com/scala/collection-strawman/compare/master...olafurpg:scalafix)
+(available soon) that takes care of changing the imports of an existing project to use
+the `strawman` namespace. That should automate most of the adaptation work.
```
Comment (heathermiller) [inline blog/_posts/2017-06-30-collections-0.2.0.md]: More detail here. Expand.

File: blog/_posts/2017-06-30-collections-0.2.0.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+## Next steps
+
+From a user point of view, little should have changed, even though internally, a lot
```
Comment (heathermiller) [inline blog/_posts/2017-06-30-collections-0.2.0.md]: "little should have changed"? What do you mean? If interface, say it.

File: blog/_posts/2017-06-30-collections-0.2.0.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+has changed (we will write more about that in a future post)! We encourage you
```
Comment (heathermiller) [inline blog/_posts/2017-06-30-collections-0.2.0.md]: Don't include this paragraph unless you're actually going to include some information.

> "(we will write more about that in a future post)"

Can you give a summary? There's little information in this entire sentence.

File: blog/_posts/2017-06-30-collections-0.2.0.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+to give a try to the new collections and to
+[report](https://github.com/scala/collection-strawman/issues) any issue you encounter.
+
+From now on, we will cut new releases every month. The goals of the
```
Comment (heathermiller) [inline blog/_posts/2017-06-30-collections-0.2.0.md]: Maybe this should be earlier, since this is the real goal of the release, right?

Comment (julienrf) [inline blog/_posts/2017-06-30-collections-0.2.0.md]: No, the goal of the release is to provide an artifact that people can use.

**Pull request merged**